src/containers/Tenant/Diagnostics/TenantOverview/TenantOverview.scss

```diff
@@ -30,13 +32,15 @@
         align-items: center;
 
         margin-bottom: 10px;
+
+        line-height: 24px;
     }
 
     &__top-label {
-        margin-bottom: 20px;
+        margin-bottom: detailedOverview.$section-title-margin;
 
         font-weight: 600;
-        line-height: 24px;
+        line-height: detailedOverview.$section-title-line-height;
```

Comment:
variables introduced to explicitly state that these sizes should match between the overview and the healtchcheck blocks, to align the contents

Reply:
today I learned something new :). Namely, sass variables scoping.